semgrep/semgrep/formatter/text.py

```diff
@@ -130,16 +130,14 @@ def _build_text_timing_output(
             rule["id"]: functools.reduce(
                 lambda x, y: (x[0] + y[0], x[1] + y[1]),
                 (
-                    (t["run_times"][i] - t["parse_times"][i], t["match_times"][i])
```

Comment:
@emjin not sure what the intention is here. Is it important that the output is run_time - parse_time?

Reply:
The run_time includes the time it took the file to parse, which, because it varies based on whether the file is cached or not, we don't want to count

Reply:
Oh I see. I'll put some of this back into profiler then. Didn't consider this :D